docs/UserGuide.md

```diff
@@ -356,30 +356,32 @@ Examples:
                This means that if Prudy has 2 clients: <code>Alex</code> and <code>Bernice</code> given in that order, and you did <code>find-client n/bernice</code> to filter out <code>Alex</code>. An INDEX of <code>1</code> will refer to <code>Bernice</code> instead of <code>Alex</code>.
   </box>
 
-### 5.2.4 Editing a Client’s Details: `edit`
+### 5.2.4 Editing a Client’s Details: `edit-client`
```

Comment:
Can change this to "Editing a Client" to standardise. The table of content also needs to be changed through this edit